src/frontends/tensorflow/tests/convert_tricky_models.cpp

```diff
@@ -246,3 +246,19 @@ TEST_F(TransformationTestsF, DISABLED_ModelWithDilatedGroupConvolution) {
         model_ref = make_shared<Model>(OutputVector{transpose_after}, ParameterVector{x});
     }
 }
+
+TEST_F(TransformationTestsF, ModelWithSaveV2) {
```

Comment:
Does this test cover UnsupportedConstToResultRemover transform?

Reply:
Yes, it includes both SaveV2 and StringConst->Identity case.